src/components/editor/Editor.tsx

```diff
@@ -21,7 +21,7 @@
 
 import * as React from "react";
 import { Project, File, Directory, FileType, languageForFileType } from "../../model";
-import { View, EditorPane } from "./EditorPane";
+import { EditorPane, View } from ".";
```

Comment:
Circular reference between, which might cause a confusion about location of the EditorPane and View. Can you use `import { EditorPane } from "./EditorPane";` and `import { View } from "./View";` instead?